### ISSUE TITLE
auto-hide.css breaks Firefox 72 entirely

### ISSUE THREAD

**Issue by LOuroboros** [Timvde/UserChrome-Tweaks#162]
There isn't much else to say, really.
[auto-hide.css](https://github.com/Timvde/UserChrome-Tweaks/blob/master/toolbars/auto-hide.css) breaks Firefox 72 entirely. The browser is not really usable with it alone.
https://streamable.com/efz9t

**Comment by Timvde** [comment]
I just merged #160. This will now only hide the location bar and not the bookmarks toolbar.

**Comment by LOuroboros** [comment]
It seems to render the bookmar bar unusable, @Timvde.
https://streamable.com/ecm78

By the way, is there a separate code to auto-hide the bookmarks bar then, now that it's not included in auto-hide.css?
I'll take a look again, but I couldn't find one.

**Comment by Timvde** [comment]
Hmm, that's pretty useless indeed. In #160, @matt-h links to https://github.com/MrOtherGuy/firefox-csshacks/blob/d6a1a8663dc91db9a72b5fce0e61cf30f22e59e7/chrome/autohide_bookmarks_toolbar.css, does that work for you? I don't know if they can work together.

**Comment by LOuroboros** [comment]
> does that work for you?

Not really. The bookmarks bar is still unusable.

https://streamable.com/yj8gx

**Comment by Timvde** [comment]
Hmm, that's what I feared... I'm not good enough with CSS to fix this myself. I'm sorry.

**Comment by matt-h** [comment]
This seemed to work fairly well for me for hiding both the navbar and  bookmark bar at once.

```
#PersonalToolbar:not([customizing]),
#nav-bar:not([customizing="true"]):not([inFullscreen]) {
	min-height: 1px !important;
	max-height: 0px !important;
	margin-top: 1px !important;
	margin-bottom: -1px !important;
	transition: all 50ms linear 0s !important;
	z-index: -5 !important;
}

#navigator-toolbox:hover:not([inFullscreen]) :-moz-any(#nav-bar),
#navigator-toolbox:focus-within :-moz-any(#nav-bar) {
	min-height: 32px !important;
	max-height: 32px !important;
	margin-top: 1px !important;
	margin-bottom: -32px !important;
	transition: all 50ms linear 0s !important;
	z-index: 5 !important;
}

#navigator-toolbox:hover:not([inFullscreen]) :-moz-any(#PersonalToolbar),
#navigator-toolbox:focus-within :-moz-any(#PersonalToolbar) {
	min-height: 32px !important;
	max-height: 32px !important;
	margin-top: 32px !important;
	transition: all 50ms linear 0s !important;
	z-index: 5 !important;
}
```

**Comment by sans-c** [comment]
> This seemed to work fairly well for me for hiding both the navbar and bookmark bar at once.

  Yep, that works for me as well but reintroduces the content shifting. Also the bookmark bar seems a little bigger than before for me. For people who use the bookmark toolbar, its a working solution at none the less

**Comment by LOuroboros** [comment]
> This seemed to work fairly well for me for hiding both the navbar and bookmark bar at once.

That code does seem to work perfectly on my end!

https://streamable.com/3djau

**Comment by willbelr** [comment]
A little out topic, but as I could not find a satisfying solution, I went for this alternative https://addons.mozilla.org/en-CA/firefox/addon/bookmark-toolbar-menu-button/. Hope this helps
